File: Beecrowd/1074 Par ou Ímpar/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        int n, x;

        n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            x = sc.nextInt();

            if (x == 0)
                System.out.println("NULL");

            if (x < 0) {
                if (x % 2 == 0)
                    System.out.println("EVEN NEGATIVE");
                else
                    System.out.println("ODD NEGATIVE");
            }
            if (x > 0) {
                if (x % 2 == 0)
                    System.out.println("EVEN POSITIVE");
                else
                    System.out.println("ODD POSITIVE");
            }
        }

        sc.close();
    }
}

File: Beecrowd/1074 Par ou Ímpar/main.cpp
#include <iostream>
 
using namespace std;
 
int main() {
 
	int N, i, X;
	
	cin >> N;
	
	for(i = 0; i < N; i++){
		cin >> X ;
	
		if(X == 0)
			cout << "NULL" << endl;
		
		if(X < 0){
			if(X % 2 == 0)
				cout << "EVEN NEGATIVE" << endl;
			else
				cout << "ODD NEGATIVE" << endl;
		}
		if(X > 0){
			if(X % 2 == 0)
				cout << "EVEN POSITIVE" << endl;
			else
				cout << "ODD POSITIVE" << endl;
		}
	}
 
    return 0;
}